src/types/state.ts

```diff
@@ -98,7 +98,6 @@ export type CompareResultsItem = {
   delta_percentage: number;
   magnitude: number;
   is_improvement: boolean;
-  is_empty: boolean;
```

Comment:
I removed this field from the response in https://github.com/mozilla/treeherder/pull/8216